portal-web/test/functional/com/liferay/portalweb/tests/enduser/headless/HeadlessDeliveryAPI/HeadlessDeliveryStructuredContent.testcase

```diff
@@ -46,11 +88,12 @@ definition {
 		task ("Given a web content are created with default priority") {
 			var ddmStructureId = WebContentStructures.getDdmStructureId(structureName = "content-structure");
 
-			var responseToParse = HeadlessWebcontentAPI.createStructuredContentDraft(
+			var responseToParse = HeadlessWebcontentAPI.createStructuredContentWithPriority(
 				data = "<p>My content</p>",
```

Comment:
looks good, but we may want to modify the method createStructureContentWithPriority. In this particular test we want the default value of the priority, so, it's about not setting it. Take a look how Mateus handled it [here](https://github.com/liferay-frontend/liferay-portal/blob/bb9da532e57ebef08ecbef7c6eaff56e9f33d4a7/portal-web/test/functional/com/liferay/portalweb/macros/headless/journal/HeadlessWebcontentAPI.macro#L161)